plugins/polio/js/src/components/campaignCalendar/popper/RoundPopper.js

```diff
@@ -81,13 +83,19 @@ const RoundPopper = ({
                             <FormattedMessage {...MESSAGES.raStatus} />:
                         </Grid>
                         <Grid item sm={6} container justifyContent="flex-start">
-                            {campaign.original.risk_assessment_status}
+                            {formatMessage(
+                                MESSAGES[
+                                    campaign.original.risk_assessment_status
+                                ],
+                            ) ?? campaign.original.risk_assessment_status}
```

Comment:
nope,, this will create a warning or an error if MESSAGE[campaign...]) does not exists